app/templates/test.twig

```diff
@@ -1,15 +1,11 @@
 {% extends "layouts/basic.layout.twig" %}
 
-{% block title %}Index{% endblock %}
+{% block title %}{{title}}{% endblock %}
```

Comment:
Here, we don't need a variable for 'Title', we can just set it, _unless_ the title will changed dynamically, say to include the user's name